docs/xtrace.md

```diff
@@ -241,35 +246,108 @@ The following table lists the possible Java components (`<component>`). To inclu
 
 :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** When specifying the **mt** component you must also specify the `methods` option.
 
 The following table lists all the tracepoint groups (`<group>`). Each group is associated with one or more Java components:
 
-|Component name or names     |  Group name       |  Description                                                                                                              |
-|----------------------------|-------------------|---------------------------------------------------------------------------------------------------------------------------|
-|**j9mm**                    | `gclogger`        | A set of tracepoints that record each garbage collection cycle. Equivalent to `-verbose:gc` output                        |
-|**j9prt**                   | `nlsmessage`      | A set of tracepoints that record each NLS message that is issued by the VM.                                               |
-|**j9jcl**, **j9vm**         | `verboseclass`    | A set of tracepoints that record each class as it is loaded. Equivalent to `-verbose:class` output.                       |
-|**j9jni**, **j9vm**         | `checkjni`        | A set of tracepoints that record JNI function checks. Equivalent to `-Xcheck:jni` output.                                 |
-|**j9vm**                    | `checkmemory`     | A set of tracepoints that record memory checks. Equivalent to `-Xcheck:memory` output.                                    |
-|**j9vm**                    | `checkvm`         | A set of tracepoints that record VM checks. Equivalent to `-Xcheck:vm` output.                                            |
-|**j9jit**                   | `verbose`         | A set of tracepoints that record JIT compiler configuration and method compilation. Equivalent to `-Xjit:verbose` output. |
-|**mt**                      | `compiledMethods` | A set of tracepoints that record compiled Java methods.                                                                   |
-|**mt**                      | `nativeMethods`   | A set of tracepoints that record Java native methods.                                                                     |
-|**mt**                      | `staticMethods`   | A set of tracepoints that record Java static methods.                                                                     |
+|Component name or names     |  Group name            |  Description                                                                                                              |
+|----------------------------|------------------------|---------------------------------------------------------------------------------------------------------------------------|
+|**j9mm**                    | `adaptivethread`       | A set of tracepoints that record GC adaptive threading.                                                                   |
```

Comment:
In `j9prt.tdf`, I found one group not listed here:
```
TraceExit=Trc_PRT_j9hypervisor_vendor_init_exit Group=j9mem Overhead=1 Level=5 NoEnv Template="j9hypervisor_vendor_init exited with ret=%d"
```
That tracepoint is used in `j9hypervisor_common.c`.

Reply:
How about I change the group for this tracepoint to j9hypervisor, it seems an error for it to be j9mem. Also a group with a single tracepoint isn't that useful.

Reply:
https://github.com/eclipse-openj9/openj9/pull/20220

Reply:
Agreed.